rad-exe/Cargo.toml

```diff
@@ -0,0 +1,17 @@
+[package]
+name = "rad-exe"
+version = "0.1.0"
+authors = ["Fintan Halpenny fintan.halpenny@gmail.com"]
+edition = "2018"
+license = "GPL-3.0-or-later"
+
+[lib]
+doctest = true
+test = false
+
+[dependencies]
+anyhow = "1.0"
+structopt = "0.3"
```

Comment:
![clueless](https://i.imgflip.com/5l8r7d.jpg)

Reply:
![clap](https://media2.giphy.com/media/l3q2XhfQ8oCkm1Ts4/200.gif)

Reply:
With an endorsement from Morgan Freeman, what could go wrong?

Reply:
What's the reasoning behind using structopt rather than clap (which includes structopt these days)?

Reply:
The plan would be to use `clap-3.0.0` once it's out of its beta phase. Would you prefer to rely on `clap-3.0.0-beta.4`?

Reply:
Ah, yeah I was misled by the documentation on clap.rs into thinking that clap 3.0 was out of beta. Structopt makes sense then.

Reply:
Sweet :grin: